### ISSUE TITLE
geom_candlestick: when open and close prices are equal, no candle body.

### ISSUE THREAD

**Issue by githubuser2500114** [business-science/tidyquant#166]
As shown in the plot below, when the open and close prices are equal, the (thin) candle body is missing.
Is this on purpose? 
![image](https://user-images.githubusercontent.com/12764596/80319609-9ee74680-8811-11ea-82e2-f67455627a97.png)

**Comment by mdancho84** [comment]
Are these trading days? Jan 04 and Jan 05 are likely weekends. If Jan 06 is a trading day, which has both a high and a low, then if open = close, the box will have no height.

**Comment by githubuser2500114** [comment]
Yes, Jan 04 and Jan 05 are not trading days. My concern is that on Jan 06, although open = close, is it customary to use a thin line to indicate the open/close prices?
